lib/vanagon/platform/windows.rb

```diff
@@ -1,6 +1,7 @@
 class Vanagon
   class Platform
     class Windows < Vanagon::Platform
+      attr_accessor :proj_root
```

Comment:
Yeah, We can leave the dependency to the project object there. I am currently working on a work around for the tests so we don't have to have this.